### ISSUE TITLE
Noise from repeater output suddenly increase half way duing QSO

### ISSUE THREAD

**Issue by fly-ing-otter** [rolandturner/9v1rs#6]
Good afternoon Roland

For the past months it have been very consistent and many a times that half way during the QSO, the repeater output will suddenly be very noisy. This makes QSO almost impossible and either we got to QRT to hop on to repeaters from our neighboring country.

Many a times the increase in the noise was when someone is still talking on air and we can hear the obvious sudden hissing. This rules out local QRM at the transmitting/receiving station as it seems that the repeater settings was changed intentionally. This sometimes forces me to go on reverse to pick up stations that are only near my QTH.

I am not sure what is happening to the repeater as the power required to trigger the repeater and having acceptable readability on the receiving end has been increasing. For instance, in September 2020, I only require 0.5 W to make a QSO via the repeater. This increases to 2.5 W, to 5.5 W and now to 7 W for a proper QSO to be made. There are days where 7 W is insufficient as well when the repeater has some issue. I have checked my station and found nothing that was wrong be it antenna or power.

I hope that the Repeater Community will be able to resolve the issue or come to a conclusion with such matter as it has been happening for months consistently and not a one time off issue.

I thank you and your Team for the help and support with regard to this issue.

Cheers.

Best Regards
Kevin Tan
9V1KT

**Comment by rolandturner** [comment]
There are certainly some problems that I can't offer a hypothesis for. I suspect that the current problem is cavities out of tune combined with returning the station to its rated power last week meaning that the AGC is being flooded. I'll work with Jeff shortly to get the tuning checked, corrected if necessary, and documented.

The broader concerns that you are describing are plain weird unless the cavities or radio are both continually degrading, which seems unlikely, however you're not the only person reporting problems of this kind.

I suspect that failing to designate a formal reporting and rectification channel (this one!) until last week hasn't helped. Hopefully more accurate and timely documentation of what people are seeing will allow us to resolve the issues and get to a reliable set up sooner rather than later!

**Comment by fly-ing-otter** [comment]
Yeah is kind of weird that the required amount of power needed increases with time. As mentioned previously, now I require 7 W to go on and hold a QSO. However, 0.5 W would still be able to at least trigger the repeater feedback. Today, I just realised that 0.5 W was unable to at least trigger the repeater at all. I guess this was due to the recent heavy rain which may cause water to seep into some kind of repeater equipment.

**Comment by rolandturner** [comment]
The change in behaviour in the minutes before the net on the 3rd would appear consistent with the idea that rain water is involved (it had just started raining at the site). If so, we'd expect periodic random changes of this type, which is what we're seeing, and the short-term-observable changes would always be in the one direction (water arrives quickly but evaporates slowly).

The sudden increase in noise is almost certainly more TX broadband noise power bleeding through, causing the RX AGC to wind itself down (so we hear more noise and less signal). Quite how rainfall might be having this effect is not yet clear to me.
